content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,254 @@
+
+That triggers volume binding and/or provisioning, either immediately if
+the storage class uses immediate volume binding or when the pod is
+tentatively scheduled onto a node (`WaitForFirstConsumer` volume
+binding mode). The latter is recommended for generic ephemeral volumes
+because then the pod scheduler is free to choose a suitable node for
+the pod. With immediate binding, it is forced to use a node that has
+access to the volume once it is available.
+
+These additional PVCs are owned by the pod. When the pod gets deleted,
+the Kubernetes garbage collector deletes the PVC, which then usually
+triggers deletion of the volume because the default reclaim policy of
+storage classes is to delete volumes. If for some reason an ephemeral
+volume is not meant to be deleted, a storage class with "retain" as
+reclaim policy can be used.
+
+Once these PVCs exist, they can be used like any other PVC. In
+particular, they can be referenced as data source in volume cloning or
+snapshotting. The PVC object also holds the current status of the
+volume.
+
+### PVC Naming
+
+Naming of the additional PVCs is currently deterministic: the name is
+a combination of pod name and volume name, with a hyphen (`-`) in the
+middle. In the example above, the PVC name will be
+`my-app-scratch-volume`.  This deterministic naming makes it easier to
+interact with the PVC because one does not have to search for it once
+the pod name and volume name are known.
+
+However, it also introduces a potential conflict between different
+pods (a pod "pod-a" with volume "scratch" and another pod with name
+"pod" and volume "a-scratch" both end up with the same PVC name
+"pod-a-scratch") and between pods and manually created PVCs.
+
+Such conflicts are detected: a PVC is only used for an ephemeral
+volume if it was created for the pod. This check is based on the
+ownership relationship. An existing PVC is not overwritten or
+modified. But this does not resolve the conflict because without the
+right PVC, the pod cannot start.
+
+Therefore care must be taken when naming pods and volumes inside the
+same namespace such that these conflicts cannot occur.
+
+### Security
+
+Enabling the GenericEphemeralVolume feature allows users to create
+PVCs indirectly if they can create pods, even if they do not have
+permission to create them directly. Cluster administrators must be
+aware of this. If this does not fit their security model, they have
+two choices:
+- Explicitly disable the feature through the feature gate, to avoid
+  being surprised when some future Kubernetes version enables it
+  by default.
+- Use a [Pod Security
+  Policy](/docs/concepts/policy/pod-security-policy/) where the
+  `volumes` list does not contain the `ephemeral` volume type.
+
+The normal namespace quota for PVCs in a namespace still applies, so
+even if users are allowed to use this new mechanism, they cannot use
+it to circumvent other policies.
+
+## {{% heading "whatsnext" %}}
+
```

Comment:
Please also signpost readers to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage

Reply:
Added:
```
### Ephemeral volumes managed by kubelet

See [local ephemeral storage](/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage).
```